test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -1068,3 +1051,21 @@ Describe "WSMan Config Provider tab complete tests" -Tags Feature,RequireAdminOn
         # TODO: move to test cases above once working
     }
 }
+
+Describe "TabCompletion tests requiring elevation" -Tags Feature,RequireAdminOnWindows {
+
+    It 'Should complete about help topic' {
+
+        $aboutHelpPath = Join-Path $PSHOME (Get-Culture).Name
+
+        ## If help content does not exist, tab completion will not work. So update it first.
+        if (-not (Test-Path (Join-Path $aboutHelpPath "about_Splatting.help.txt")))
+        {
+            Update-Help -Force -ErrorAction SilentlyContinue
+        }
+
+        $res = TabExpansion2 -inputScript 'get-help about_spla' -cursorColumn 'get-help about_spla'.Length
+        $res.CompletionMatches.Count | Should Be 1
```

Comment:
The same - maybe `Should BeGreaterThan 0`.

Reply:
Fixed.